domain/service/service.go

```diff
@@ -120,30 +120,6 @@ type Service struct {
 	datastore.VersionedEntity
 }
 
```

Comment:
Note: This was [moved to servicestatemanager.go](#diff-c8e81cdb91f28c802e17dec1fc06248d)